posting/list.go

```diff
@@ -1191,6 +1193,8 @@ func (l *List) encode(out *rollupOutput, readTs uint64, split bool) error {
 		out.parts[startUid] = plist
 	}
 
+	out.plist.Bitmap = codec.ToBytes(bm)
```

Comment:
Need to figure out why things were working without this change.

Reply:
Add a comment here, we don't set plist here. It is set by the finalize() function at the end.